docs/extend/plugins.md

```diff
@@ -53,6 +53,10 @@ The following plugins exist:
   another volume plugin that provides multi-host volumes management for Docker
   using GlusterFS.
 
+* The [Horcrux Volume Plugin](https://github.com/muthu-r/horcrux)
+  An On Demand,Version controlled access to your Data. Currently supports scp, minio, aws s3 backends.
+  More coming soon. Its open source and written in Go.
```

Comment:
I see some nits in there, perhaps;

```Markdown
The [Horcrux Volume Plugin](https://github.com/muthu-r/horcrux) allows on-demand,
version controlled access to your data. Horcrux is an open-source plugin, 
written in Go, and supports SCP, [Minio](https://www.minio.io), and Amazon S3.
```

(I removed "currently" and "coming soon", because documentation should not try and look into the future :))